src/fsharp/build.fs

```diff
@@ -3996,16 +4059,21 @@ type TcImports(tcConfigP:TcConfigProvider, initialResolutions:TcAssemblyResoluti
         tcImports.RegisterCcu(ccuinfo);
         let phase2 () = 
 #if EXTENSIONTYPING
-            ccuinfo.TypeProviders <- tcImports.ImportTypeProviderExtensions (tpApprovals, displayPSTypeProviderSecurityDialogBlockingUI, tcConfig, filename, ilScopeRef, ilModule.ManifestOfAssembly.CustomAttrs.AsList, ccu.Contents, invalidateCcu, m)
+            ccuinfo.TypeProviders <- tcImports.ImportTypeProviderExtensions 
+#if TYPE_PROVIDER_SECURITY
```

Comment:
in master yes ... I ended up reproducing the removal here.  I am actually not really looking forward to the merge.  I will make a pass theat removes everything inside these if defs before I do the merge.